pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -585,6 +589,130 @@ func TestPreFilterPlugins(t *testing.T) {
 
 }
 
+func TestPlugins(t *testing.T) {
+	tests := []struct {
+		name     string
+		plugins  []*TestPlugin
+		wantCode Code
+	}{
+		{
+			name: "SuccessFilter",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj:  injectedResult{FilterStatus: int(Success)},
+				},
+			},
+			wantCode: Success,
+		},
+		{
+			name: "ErrorFilter",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj:  injectedResult{FilterStatus: int(Error)},
+				},
+			},
+			wantCode: Error,
+		},
+		{
+			name: "UnschedulableFilter",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj:  injectedResult{FilterStatus: int(Unschedulable)},
+				},
+			},
+			wantCode: Unschedulable,
+		},
+		{
+			name: "UnschedulableAndUnresolvableFilter",
+			plugins: []*TestPlugin{
+				{
+					name: "TestPlugin",
+					inj: injectedResult{
+						FilterStatus: int(UnschedulableAndUnresolvable)},
+				},
+			},
+			wantCode: UnschedulableAndUnresolvable,
+		},
+		// followings tests cover multiple-plugins scenarios
+		{
+			name: "ErrorAndSuccessFilters",
```

Comment:
This is actually "ErrorAndError"

We should also have "SuccessAndSuccess", and "ErrorAndSuccess".